content/runbooks/node/NodeFilesystemAlmostOutOfFiles.md

```diff
@@ -23,6 +23,11 @@ of the cluster.
 
 ## Diagnosis
 
+- Check the current inode usage by running the command df -i. This will show you the inode usage for all mounted file systems. Identify the file system(s) that are running low on inodes.
```

Comment:
```suggestion
- Check the current inode usage by running the command `df -i`. This will show you the inode usage for all mounted file systems. Identify the file system(s) that are running low on inodes.
```